content/en/APIUseCases/create_asset.md

```diff
@@ -0,0 +1,229 @@
+---
+title: "Use Our API to Create an Asset"
+linkTitle: "Create an Asset"
+weight: 110
+description: >
+  Use this sequence of REST calls to create an asset.
+---
+
+{{% pageinfo %}}
+Use this document with our [Cobalt API documentation](https://docs.cobalt.io) to
+manage assets and pentests.
+{{% /pageinfo %}}
+
+<!-- Future task: set up variables for `YOUR-PERSONAL-API-TOKEN` and
+`YOUR-V2-ORGANIZATION-TOKEN`. May support automated populating of REST calls. -->
+
+This page describes how you can use our API To create an [asset](../../getting-started/glossary/#asset). 
+
+## Create an API Token in the Cobalt UI
+
+1. Navigate to https://app.cobalt.io/settings/api-tokens.
+1. If needed, sign in to the app.
+1. Enter an API Token Name.
+1. Select Generate New Token.
+1. In the modal that appears, you should see your API Token, in the **Secret
+   Token** text box.
+
+Save the API Token. After you exit the modal, you won't see the full token again.
+If you lose it, you may have to revoke the token and start over.
+
+Substitute the API Token for `YOUR-PERSONAL-API-TOKEN` in the REST calls
+described on this page.
+
+## Use the API Token to Authorize Access 
+
+Next, you can use the API Token to authorize access to our other endpoints. Take
+the API Token that you [generated](#create-an-api-token-in-the-cobalt-ui). Substitute that value for `YOUR-PERSONAL-API-TOKEN`:
```

Comment:
Next, you can use the API token to authorize in the Cobalt API. Take
the API token that you [generated](#create-an-api-token-in-the-cobalt-ui). Substitute that value for `YOUR-PERSONAL-API-TOKEN`:

Reply:
FWIW, when I disable 2FA for an account, I've found REST calls to sign in and create an API token. But, those REST calls are not part of the Cobalt API. So I'm going to change this to 

...

Next, you can use the API token to authorize access to the Cobalt API. Take
the API token that you [generated](https://github.com/cobalthq/cobalt-product-public-docs/pull/145/files#create-an-api-token-in-the-cobalt-ui). Substitute that value for YOUR-PERSONAL-API-TOKEN: